src/languages/mathematica.js

```diff
@@ -55,17 +55,18 @@ export default function(hljs) {
 
   const NUMBERS = {
     className: 'number',
-    relevance:0,
+    relevance: 0,
     begin: MATHEMATICA_NUMBER_RE,
   };
 
   const SYMBOL_RE = /[a-zA-Z$][a-zA-Z0-9$]*/;
   const SYMBOLS = {
     begin: SYMBOL_RE,
+    className: 'symbol',
```

Comment:
You brought it back... intentional?

Reply:
<img width="293" alt="Screen Shot 2020-11-01 at 9 27 40 AM" src="https://user-images.githubusercontent.com/6473/97805498-84f54900-1c24-11eb-930c-bb9b5fb38226.png">

We really don't want this kind of nesting... either it's a symbol or a keyword, not both.

Reply:
Technically, in Mathematica, it _is_ both. Symbols are symbols. Some have a built-in definition, some don't.

That was just a small note, I won't comment on whether it should be considered both with highlight.js, as that's a different question.

Reply:
Even if technically true this isn't how we'd represent it... :-)  It's either a `symbol` or `builtin-symbol` (the original names given here before aliasing) and the mode should return a single one of those to wrap the symbol, not nested matchers.

Reply:
If someone wanted to style symbols bold, and built-in symbols red, then built-in symbols should absolutely be _both_ red and bold.

Reply:
Yes, I brought it back intentionally for the reason that Szabolcs mentioned. Everything is a symbol but there are special ones that have built-in meaning. But I'm not opposed to separate the two `symbol` and `builtin-symbol` rules, however, for the love of God, I never managed to get this to work because the general `$pattern` is the same and highlight.js refuses to do this.

@joshgoebel Can you show me how to do this?

Reply:
Traditionally you'd use `beginKeywords` which under the hood is a smarter version of `regex.either`... but i fear for that here with a list of a million keywords, so we can use a callback instead.

You can't do this with `keywords` alone, it's only intended to highlight KNOWN keywords.